code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/PushTemplateConstants.kt

```diff
@@ -125,6 +125,16 @@ internal object PushTemplateConstants {
         const val CATALOG_ITEMS = "adb_items"
     }
 
+    internal object TimerKeys {
```

Comment:
nit: i like having all of the adb prefixed keys together within the PushPayloadKeys object. I am open to change though 😄

Reply:
I like your idea. 
Will keep the complete TimerKey object inside the `PushPayloadKeys`